playlists-prod/outlook.yaml

```
@@ -701,56 +714,43 @@
   group: Other Item APIs
   api_set:
     Mailbox: '1.1'
-- id: outlook-calendar-properties-apis
-  name: Work with calendar properties APIs (Compose)
-  fileName: calendar-properties-apis.yaml
-  description: >-
-    Gets and sets isAllDayEvent and sensitivity properties of an appointment in
-    Compose mode.
-  rawUrl: >-
-    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/outlook/99-preview-apis/calendar-properties-apis.yaml
-  group: Preview APIs
-  api_set:
-    Mailbox: preview
-- id: outlook-item-body-prepend-text-on-send
-  name: Prepend text to item body on send
-  fileName: prepend-text-on-send.yaml
-  description: >-
-    Prepends text to the beginning of the message or appointment's body once
-    it's sent.
-  rawUrl: >-
-    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/outlook/99-preview-apis/prepend-text-on-send.yaml
-  group: Preview APIs
-  api_set:
-    Mailbox: preview
-- id: outlook-sensitivity-labels-sensitivity-labels-catalog
-  name: Work with the sensitivity labels catalog
-  fileName: sensitivity-labels-catalog.yaml
+- id: outlook-other-item-apis-work-with-client-signatures
+  name: Work with client signatures (Compose)
+  fileName: work-with-client-signatures.yaml
   description: >-
-    Determines if sensitivity labels are enabled on the mailbox and retrieves
-    all available labels from the catalog.
+    Check if the client signature is enabled, disable the client signature, get
+    the compose type, and set a signature in Compose mode.
```

Consider updating the snippet in a subsequent PR
```suggestion
    Checks if the client signature is enabled, disables the client signature, gets
    the compose type, and sets a signature in Compose mode.
```

---

Thanks. Made the suggested changes in this PR, since they're minor.